.firebaserc

```diff
@@ -1,17 +1,17 @@
 {
   "projects": {
-    "default": "kai-ai-f63c8",
+    "default": "marvel-project-a0cc2",
```

Comment:
let the default be "default": "marvel-onboarding"

Reply:
will do